azure-pipelines/release.yml

```diff
@@ -44,8 +47,15 @@ stages:
               npm install --global vsce
             displayName: 'Install vsce'
           - pwsh: |
+              $artifactFolderPrefix = "VSIX_Release_"
+              if ("${{ parameters.uploadPrerelease }}" -eq "true") {
+                $artifactFolderPrefix = "VSIX_Prerelease_"
+              }
+
+              Write-Host "Using artifacts from $artifactFolderPrefix"
+
               # Our build pipeline would generated build based on attempt number. Publishing the latest attempt.
-              $allArtifacts = Get-ChildItem -Path "VSIXs - Attempt*" | Sort-Object -Descending
+              $allArtifacts = Get-ChildItem -Path "$artifactFolderPrefix*" | Sort-Object -Descending
```

Comment:
Hmm, this is doing a textual sort right, so pray we never have 10 attempts?

Reply:
Yeah.  That's a pre-existing issue though